data/json/vehicles/carts.json

```diff
@@ -97,9 +97,48 @@
       { "x": -1, "y": 1, "part": "wheel_small" }
     ],
     "items": [
-      { "x": -1, "y": 0, "chance": 5, "items": [ "towel" ] },
-      { "x": -1, "y": 0, "chance": 10, "items": [ "golf_club" ] },
-      { "x": -1, "y": 1, "chance": 8, "items": [ "golf_club" ] }
+      { "x": -1, "y": 0, "chance": 25, "repeat": 2, "item_groups": [ "golf_cart" ] },
+      { "x": -1, "y": 1, "chance": 25, "repeat": 2, "item_groups": [ "golf_cart" ] }
+    ]
+  },
+  {
+    "id": "golf_cart_4seat",
+    "type": "vehicle",
+    "name": "Golf Cart",
+    "blueprint": [
+      [ "o|o" ],
+      [ "o|o" ]
+    ],
+    "parts": [
+      { "x": 0, "y": 0, "part": "frame_horizontal" },
+      { "x": 0, "y": 0, "part": "seat" },
+      { "x": 0, "y": 0, "part": "roof" },
+      { "x": 0, "y": 0, "part": "engine_electric" },
+      { "x": 0, "y": 0, "part": "controls" },
```

Comment:
Add a dashboard, or one wouldn't be able to control electrics of the car.

Reply:
I tested both the original and my new golf cart and both operate just fine. Also I made the other changes.